akka-javasdk/src/main/scala/akka/javasdk/impl/serialization/ThrowableMixin.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2021-2025 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.javasdk.impl.serialization
+
+import scala.annotation.nowarn
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+
+@nowarn("msg=never used")
+abstract class ThrowableMixin {
+  @JsonIgnore
+  private var detailMessage: String = _
+
+  @JsonIgnore
+  private var cause: Throwable = _
+
+  @JsonIgnore
+  private var stackTrace: Array[StackTraceElement] = _
```

Comment:
It might be worth to add a test demonstrating that stack traces are not generated. 

Additionally, if users specify a `cause` or wrap another exception, the stack trace of the `cause` should also not be serialized.

Reply:
Actually, this test is good enough, because you can't set the cause when implementing a custom exception right now.

Reply:
Yeah, but then we know that the same rules will be applied to any Throwable. If you think it's still worth testing it, I could add one more.